### ISSUE TITLE
Can't access bioconductor website or use BiocManager::install

### ISSUE THREAD

**Issue by Tommy0398** [Bioconductor/bioconductor.org#248]
Hi,

When I try to access bioconductor to download a package I get the error `This site can’t be reached`, however if I prefix the webid with www. it works. E.g. https://bioconductor.org/packages/DESeq2 into https://www.bioconductor.org/packages/DESeq2.

I also recently have been unable to download packages with `BiocManager::install` where I've been able to fine before now, with it giving me this error: 
```
Bioconductor version cannot be validated; no internet connection?  See #troubleshooting section in vignette
Warning messages:
1: package ‘BiocManager’ was built under R version 4.2.3 
2: In file(con, "r") :
  URL 'https://bioconductor.org/config.yaml': status was 'SSL connect error'
3: In readLines(...) : line 6 appears to contain an embedded nul
4: In readLines(...) :
  incomplete final line found on 'http://bioconductor.org/config.yaml'

> BiocManager::install("GEOquery")
Error: Bioconductor version cannot be validated; no internet connection?  See #troubleshooting section in vignette
```

I've seen a few forums mentioning the install issue, but I didn't see any where it suddenly stopped working or about the website. Do you have any suggestions on how I might fix this?

**Comment by LiNk-NY** [comment]
Hi @Tommy0398 
Here are some possible scenarios and troubleshooting steps:

Network Configuration Issues:

Proxy Settings: 

Check if your internet connection is using a proxy server. Proxy settings can sometimes interfere with internet access. Ensure that R and your system are configured to use the correct proxy settings.

Firewall: 

A firewall may be blocking your connection to specific websites or resources. Check your firewall settings and ensure that R and R packages have the necessary permissions.

DNS Configuration:

It's possible that your DNS (Domain Name System) settings are not resolving the URLs correctly. This could explain why adding "www" works. Try switching to a different DNS server or manually configuring your DNS settings.

SSL/TLS Issues:

The error message "SSL connect error" suggests an issue with the SSL/TLS configuration. Ensure that your R installation has the necessary SSL libraries and certificates. You might need to update or reinstall R to resolve this issue.

**Comment by LiNk-NY** [comment]
We recently made a change to have our resources use `https` by default. 
The error you are getting may be due to the SSL configuration. 

Best,
Marcel

**Comment by Tommy0398** [comment]
Hi Marcel,

Thank you for your quick response. 

I've tried disabling the firewall and nothing changed. I've also tried changing the DNS server which also didn't change anything although I'm not confident I know what I'm doing with the DNS server.

Regarding the proxy server and SSL/TLS issues I'm on a University network and this is their response regarding the issue:
University network is fully opened. There is no web filtering and port filtering only applies to specific buildings (labs etc.) and even if these are the ports that are never used by applications like R.

In case of what you are trying to access it is just https: traffic which is never blocked. I tried accessing the page in both versions from non SSD machine (same network config as yours) and had no issues doing so.

So, I'm not sure what I should try to do now. Do you have any suggestions?

Kind Regards,
Tommy

**Comment by Tommy0398** [comment]
Hi,

So I seem to have found a fix to the problem after trying to figure out what caused it. I had recently installed docker to use a program, so I uninstalled docker to see if it'd change anything and the install command is working fine now. I can also access the bioconductor website normally aswell which is odd as that has been an issue way before I downloaded docker. 

I guessing the uninstall changed something else that seems to have fixed the problem. If you have an idea of why that worked I'd be interested in understanding it, but thank you for your time the problem seems to be resolved.
